html/quote-generator.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20,500,0,-25" />
+    <link rel="stylesheet" href="../css/quote-generator.css">
+    <link rel="stylesheet" href="../css/normalize.css">
+    <title>Quote-generator</title>
+</head>
+
+<body>
+    <main class="page">
+        <div class="top-buttons">
+            <button class="top-button" id="all-authors-btn">all authors</button>
```

Comment:
This should be not a button but a link

You are moving a user to another page, which should be a separate page (you can create another html file for this)

In this case we need `a`